Automattic/jetpack
# Banner component: fix icon CSS margin offset

#### Changes proposed in this Pull Request:

* As mentioned [here](https://github.com/Automattic/jetpack/pull/21947#pullrequestreview-823838203), the icons in the `<Banner>` component are misaligned. Looks like we already pad around the icons [here](https://github.com/Automattic/jetpack/blob/ba2dd6c30be137baad96aafb2466d0aed7e182e5/projects/plugins/jetpack/_inc/client/components/banner/style.scss#L106), so the margin isn't necessary which this PR removes.

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Pull down changes to local dev & build Jetpack.
* Visit the Jetpack dashboard and look under the `Performance` section. Use the Dev Tools to toggle on Free plan if not already to observe the icons in the cards/banners there.

**Before**:

![before](https://user-images.githubusercontent.com/15803018/145649196-8cfc7499-f44f-4e6c-ab4e-71f2d1306479.png)

**After**:

![after](https://user-images.githubusercontent.com/15803018/145649209-f347bb73-7dcd-44a9-aff9-bb2514500581.png)
